tests/neg/cannot-reduce-inline-match.check

```diff
@@ -1,11 +1,9 @@
 -- Error: tests/neg/cannot-reduce-inline-match.scala:3:13 --------------------------------------------------------------
-3 |    inline x match { // error
-  |           ^
-  |           cannot reduce inline match with
-  |            scrutinee:  {
-  |             "f"
-  |           } : String("f")
-  |            patterns :  case _:Int
-  | This location is in code that was inlined at cannot-reduce-inline-match.scala:9
-4 |      case _: Int =>
-5 |    }
+9 |  foo("f") // error
+  |  ^^^^^^^^
+  |  cannot reduce inline match with
+  |   scrutinee:  {
+  |    "f"
+  |  } : String("f")
+  |   patterns :  case _:Int
+  | This location contains code that was inlined from cannot-reduce-inline-match.scala:3
```

Comment:
This is the line which is printed at the top of the error message, so this doesn't add any information.

Reply:
The line at the top need to be updated as well

Reply:
I updated it